# MoveIt uses this configuration for controller management

moveit_controller_manager: moveit_simple_controller_manager/MoveItSimpleControllerManager
ros__parameters:
    update_rate: 1000

moveit_simple_controller_manager:
  controller_names:
    - ur5_arm_controller
    - ur5_grip_controller

  ur5_arm_controller:
    type: joint_trajectory_controller/JointTrajectoryController
    action_ns: follow_joint_trajectory
    default: true
    joints:
      - shoulder_pan_joint
      - shoulder_lift_joint
      - elbow_joint
      - wrist_1_joint
      - wrist_2_joint
      - wrist_3_joint
  ur5_grip_controller:
    type: position_controllers/GripperActionController
    joints:
      - gripper_joint1
    action_ns: gripper_cmd
    default: true